sdk/connectedvmware/arm-connectedvmware/review/arm-connectedvmware.api.md

```diff
@@ -6,58 +6,44 @@
 
 import * as coreAuth from '@azure/core-auth';
 import * as coreClient from '@azure/core-client';
+import { OperationState } from '@azure/core-lro';
 import { PagedAsyncIterableIterator } from '@azure/core-paging';
-import { PollerLike } from '@azure/core-lro';
-import { PollOperationState } from '@azure/core-lro';
-
-// @public
-export interface AvailablePatchCountByClassification {
```

Comment:
Could you find relevant swagger pr for me?

Reply:
https://github.com/Azure/azure-rest-api-specs/pull/25409